enumerables.rb

```diff
@@ -0,0 +1,135 @@
+# rubocop: disable Metrics/ModuleLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+module Enumerable
+  def my_each
+    return to_enum unless block_given?
+
+    i = 0
+    while i <= size - 1
+      yield(to_a[i])
+      i += 1
+    end
+  end
+
+  def my_each_with_index
```

Comment:
Calls block with two arguments, the item and its index, for each item in enumerable.
When a block is given when self is an array returns the array itself after calling the given block once for each element in self, similar to each method that returns the array itself.

![Screen Shot 2020-06-27 at 2 56 50 PM](https://user-images.githubusercontent.com/52430951/85931004-78cc3a00-b886-11ea-91c3-cb5b9dbdc870.png)

Reply:
![myeachindex](https://user-images.githubusercontent.com/908714/85957341-cb901980-b98c-11ea-935c-43467ceed125.png)

For what looks like the test is working. I didn't change anything. So I believe is working!